### ISSUE TITLE
Completely different behaviors between release 0.36 and 0.37

### ISSUE THREAD

**Issue by Zemax27** [mpv-player/mpv#13136]
### Important Information

Provide following Information:
- mpv version: **0.36 vs 0.37**
- Windows Version: **Windows 10 and 11**
- Source of the mpv binary: **https://sourceforge.net/projects/mpv-player-windows/files/**
- If known which version of mpv introduced the problem : **0.37 release**
- GPU model, driver and version : **Intel 630 UHD (Integrated) + NVIDIA Quadro T2000 . Drivers are up-to-date**

### Reproduction steps
I'm using two different configuration based on --vo=gpu and --vo=gpu-next
---
vo=gpu
hwdec=auto-safe
gpu-api=d3d11
gpu-context=d3d11
tone-mapping=linear
priority=high
keep-open=yes
target-prim=auto
target-trc=bt.1886

---
vo=gpu-next
hwdec=auto-safe
gpu-api=d3d11
gpu-context=d3d11
tone-mapping=bt.2446a
priority=high
keep-open=yes
target-prim=auto
target-trc=bt.1886
target-colorspace-hint=yes
vf=format:dolbyvision=no

### Expected behavior
No difference between version 0.36 and 0.37

### Actual behavior
When playing a 10bits video, the tone-mapping behaves different between the two mpv releases.
With gpu-next using either tone-mapping=bt.2446a or tone-mapping=linear behaves completely different between the two mpv releases.
When playing an 8bit video, with gpu configuration I see colored strips with version 0.37, while the playback is correct with version 0.36.

### Sample files
8bits video
https://0x0.st/HgoS.mp4

10bits video
https://0x0.st/H4eE.mp4

### LOGS

**mpv version 0.36**
gpu configuration : 
[0.36mpv-4k_SDR.txt](https://github.com/mpv-player/mpv/files/13750931/0.36mpv-4k_SDR.txt)
gpu-next configuration : 
[0.36mpv-4k_HDR_BT2446a.txt](https://github.com/mpv-player/mpv/files/13750942/0.36mpv-4k_HDR_BT2446a.txt)

**mpv version 0.37**
gpu configuration : 
[0.37mpv-4k_SDR.txt](https://github.com/mpv-player/mpv/files/13750947/0.37mpv-4k_SDR.txt)
gpu-next configuration : 
[0.37mpv-4k_HDR_BT2446a.txt](https://github.com/mpv-player/mpv/files/13750954/0.37mpv-4k_HDR_BT2446a.txt)

**Comment by Zemax27** [comment]
I add here the log files that were missing in the issue template.

**mpv version 0.36**
gpu configuration : 
[0.36mpv-4k_SDR.txt](https://github.com/mpv-player/mpv/files/13750931/0.36mpv-4k_SDR.txt)
gpu-next configuration : 
[0.36mpv-4k_HDR_BT2446a.txt](https://github.com/mpv-player/mpv/files/13750942/0.36mpv-4k_HDR_BT2446a.txt)

**mpv version 0.37**
gpu configuration : 
[0.37mpv-4k_SDR.txt](https://github.com/mpv-player/mpv/files/13750947/0.37mpv-4k_SDR.txt)
gpu-next configuration : 
[0.37mpv-4k_HDR_BT2446a.txt](https://github.com/mpv-player/mpv/files/13750954/0.37mpv-4k_HDR_BT2446a.txt)

**Comment by trump888** [comment]
when playing same H264 4k video，0.36 ok while 0.37 high gpu usage

**Comment by guidocella** [comment]
`--profile=fast` https://github.com/mpv-player/mpv/commit/703f158880

**Comment by kasper93** [comment]
> Completely different behaviors between release 0.36 and 0.37

Yes, major stable release tend to change behavior, add new features or options.

Let us know if the issue persists.
